src/frontend/src/layout/CenteredContent.tsx

```diff
@@ -1,13 +1,15 @@
 import { BackToHome } from '@/components/BackToHome'
-import { H } from '@/primitives'
+import { H, Text } from '@/primitives'
 import { Center } from '@/styled-system/jsx'
 
 export const CenteredContent = ({
   title,
+  body,
   children,
   withBackButton,
 }: {
   title?: string
+  body?: string
```

Comment:
Two things here:

1) I admit it feels a bit weird to have the possibility for both `children` and `body` here. I'd rather ditch the `children` entirely if we want to say that "any centered content should be one title + one optional body string", and update the couple components that currently pass `children`.
2) I'd rename `body` to `description` here and in other components where it was added, as I feel it better exposes the fact we want a string of text (because it's wrapped in a `p` internally), rather than any "body text".

Reply:
not sure to get the body vs description point, but will follow blindly your reco ! 
As we are early in the project, I would keep the children prop

Reply:
Actually, I've removed the body props, and kept the original one. Plus, I renamed body to description.